### PR TITLE
Add more debug log info into UI form operations

### PR DESCRIPTION
**Pull request by rochacbruno**
Add more logger DEBUG info into UI form operations.

Now UI logs should log field content filling info like this:

```bash
17:06:44 - robottelo.ui.locators - DEBUG - Accessing locator "login.username" by id: "login_login"
17:06:44 - robottelo.ui.base - DEBUG - Filling field: login.username with: CgEWJZZrNB
17:06:45 - robottelo.ui.locators - DEBUG - Accessing locator "login.password" by id: "login_password"
17:06:45 - robottelo.ui.base - DEBUG - Filling field: login.password with: changeme

```

### CODE REVIEW

File: robottelo/ui/base.py [hunk 1]
```diff
@@ -396,6 +399,7 @@ def field_update(self, loc_string, newtext):
         """
         txt_field = self.find_element(locators[loc_string])
         txt_field.clear()
+        self.logger.debug(u"Filling field: %s with: %s", loc_string, newtext)
```
Comment (sthirugn) [inline robottelo/ui/base.py]: s/Filling/Update? to match the function name?

File: robottelo/ui/base.py [hunk 2]
```diff
@@ -405,6 +409,7 @@ def text_field_update(self, locator, newtext):
         """
         txt_field = self.wait_until_element(locator)
         txt_field.clear()
+        self.logger.debug(u"Filling text field: %s with: %s", locator, newtext)
```
Comment (sthirugn) [inline robottelo/ui/base.py]: same as above

**Comment by sthirugn** [comment]
ACK with minor non blocking comments

**Comment by oshtaier** [comment]
It is preferable to use single quotes instead double ones.

**Comment by oshtaier** [comment]
ACK pending comment

**Comment by rochacbruno** [comment]
```bash
py.test -s tests/foreman/ui/test_bookmark.py -k no_query

2016-06-03 13:11:18 - robottelo.ui.browser - DEBUG - Starting container with ID "48873a39b7184a6255663d56a0927b06611a0e17dc0fa22e9895320c20e5c65e"
2016-06-03 13:11:29 - robottelo.ui.locators - DEBUG - Accessing locator "login.username" by id: "login_login"
2016-06-03 13:11:30 - robottelo.ui.locators - DEBUG - Accessing locator "login.username" by id: "login_login"
2016-06-03 13:11:31 - robottelo.ui.base - DEBUG - Updating field:login.username with:TBV6fc384f
2016-06-03 13:11:31 - robottelo.ui.locators - DEBUG - Accessing locator "login.password" by id: "login_password"
2016-06-03 13:11:32 - robottelo.ui.base - DEBUG - Updating field:login.password with:FaRrXiDw10
2016-06-03 13:11:32 - robottelo.ui.locators - DEBUG - Accessing locator "submit" by name: "commit"
...
2016-06-03 13:11:48 - robottelo.ui.base - DEBUG - Updating text field:('xpath', "//*[(@id='query' and @name='query') or (@id='bookmark_query' and @name='bookmark[query]')]") with:
2016-06-03 13:11:49 - robottelo.ui.base - DEBUG - Assigned value  to ('xpath', "//*[(@id='query' and @name='query') or (@id='bookmark_query' and @name='bookmark[query]')]")

=== 1 passed, 13 deselected in 107.97 seconds ===
```

**Pull request merged**